### ISSUE TITLE
[Feature request] handle url assets

### ISSUE THREAD

**Issue by sqal** [ktsn/vue-template-loader#11]
Basically would be nice to have this - http://vue-loader.vuejs.org/en/configurations/asset-url.html

**Comment by ktsn** [comment]
Maybe, we already can do that by chaining with html-loader?

```js
module.exports = {
  module: {
    rules: [
      {
        test: /\.html$/, 
        use: ['vue-template-loader', 'html-loader']
      }
    ]
  }
}
```

**Comment by sqal** [comment]
hmm doesn't seem to be working for me:

I added in wbepack config:

```js
{
  test: /\.html$/,
  include: paths.appSrc,
  use: [
    'vue-template-loader',
    {
      loader: 'html-loader',
      options: {
        attrs: ['img:src'],
      },
    },
  ],
},
```

Example template:

```html
<section class="container">

  <div class="card">
    <img class="logo" src="assets/img/logo_blue.svg">
  </div>

  <h5>{{ msg }}</h5>

</section>
```

But it renders only h5 tag, everthing else is stripped away

```js
var render = function() {
  var _vm = this;
  var _h = _vm.$createElement;
  var _c = _vm._self._c || _h;
  return _c('h5', [_vm._v(_vm._s(_vm.msg))]) // ?????????????
}
var staticRenderFns = []
module.exports = function(_exports) {
  var options = _exports
  if (typeof _exports === "function")
    options = _exports.options


  options.render = render
  options.staticRenderFns =
    staticRenderFns
  if (module.hot && api) {
    api.createRecord("data-v-1",
      options)
  }
  return _exports
}
var api = null
if (module.hot) {
  (function() {
    api = require(
      "vue-hot-reload-api")
    api.install(require("vue"))
    if (!api.compatible) return
    module.hot.accept()
    if (module.hot.data) {
      api.rerender("data-v-1", {
        render: render,
        staticRenderFns: staticRenderFns
      })
    }
  })()
```

**Comment by ktsn** [comment]
Ah sorry, I misunderstood html-loader behavior. I'll implement that somehow 🙂

**Comment by ktsn** [comment]
Released with [v0.2.2](https://github.com/ktsn/vue-template-loader/releases/tag/v0.2.2)
